src/lib/context-menu.ts

```diff
@@ -267,17 +268,19 @@ export class ContextMenu {
 
     const { addComment } = this.actions
```

Comment:
Why not use this `addComment` and set it's value based on `isCommentingAllowed` 

https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/master/src/views/editable_block.ts#L101

Reply:
updated